versions.props

```diff
@@ -0,0 +1,7 @@
+com.fasterxml.jackson.core:jackson-databind = 2.13.2.1
+com.google.guava:guava = 31.1-jre
+junit:junit = 4.12
```

Comment:
planning to let excavator take a first crack at junit 5 migration once this merges